.github/workflows/pull-request-test.yml

```diff
@@ -138,7 +134,6 @@ jobs:
           RENKU_RELEASE: ci-renku-${{ github.event.number }}
```

Comment:
I would move it to runner.temp as suggested by flora elsewhere

Reply:
This shows `RENKU_RELEASE` as what your comment refers to. I think you are talking about this though?
```
RENKU_VALUES_FILE: "${{ github.workspace }}/values.yaml"
```

I honestly dont know what that refers to. I dont want to break the values file any more than it is. So I would go back to this. The goal is to just get the tokens to not leak in the logs.